client/web/src/components/AvatarView/index.jsx

```diff
@@ -25,15 +26,19 @@ export function setupAvatarLipSync(audioContext, audioPlayer) {
     audioPlayer
   );
   defaultBlendshapesService_2.enableBlendshapes(externalAvatarVoiceService);
+  defaultBlendshapesService_3.enableBlendshapes(externalAvatarVoiceService);
 }
 
-const AvatarView = ({ avatarId }) => {
+const AvatarView = ({ avatarId, emotion }) => {
   const { avatarDisplay } = useAvatar({
     avatarId: avatarId,
-
+    currentEmotion: emotion,
     // Loader + Plugins
     avatarLoaders: defaultAvatarLoaders,
-    blendshapesService: defaultBlendshapesService_2,
+    blendshapesService:
+      avatarId == 'a69d8102-0285-47d6-bdf5-12d903272732'
+        ? defaultBlendshapesService_3
```

Comment:
is there a plan to eventually move all to defaultBlendshapesService_3 ? Just want to know if this condition will be around for long term

Reply:
This is a temporary solution for steve jobs avatar. We are currently working on a generic flow to handle different blendshapes, but it is still on our roadmap and might coming later a bit.

Reply:
Sounds good, I will add a TODO.